dotnet/InkRecognition/uwp-app/cs/Services/Ink/InkRecognizer.cs

```diff
@@ -1,81 +1,81 @@
-using System.Threading.Tasks;
-using Windows.Graphics.Display;
-using Windows.UI.Input.Inking;
-
-namespace Contoso.NoteTaker.Services.Ink
-{
-    public class InkRecognizer
-    {
-        List<InkRecognizerStroke> strokes;
-        InkRecognitionRoot root = null;
-
-        HttpManager httpManager;
-
-        // Use default DPI setting
-        float dpiX = 96.0f;
-        float dpiY = 96.0f;
-
-        public InkRecognizer(string appKey, string baseAddress, string destinationUrl)
-        {
-            httpManager = new HttpManager(appKey, baseAddress, destinationUrl);
-            strokes = new List<InkRecognizerStroke>();
-        }
-
-        public void AddStroke(InkStroke stroke)
-        {
-            var iaStroke = new InkRecognizerStroke(stroke, dpiX, dpiY);
-            strokes.Add(iaStroke);
-        }
-
-        public void RemoveStroke(UInt64 strokeId)
-        {
-            var strokeToRemove = strokes.Find(stroke => stroke.Id == strokeId);
-            strokes.Remove(strokeToRemove);
-        }
-
-        public void ClearStrokes()
-        {
-            strokes.Clear();
-            root = null;
-        }
-
-        public void SetDisplayInformation(DisplayInformation displayInfo)
-        {
-            // DisplayInfo.RawDpiX and DisplayInfo.RawDpiY returns 0 when monitor doesnt provide physical dimensions 
-            // or when user is in clone or multiple-monitor setup. Fallback to default DPI setting in such cases.
-            dpiX = (displayInfo.RawDpiX != 0) ? displayInfo.RawDpiX : dpiX;
-            dpiY = (displayInfo.RawDpiX != 0) ? displayInfo.RawDpiY : dpiY;
-        }
-
-        public InkRecognitionRoot GetRecognizerRoot()
-        {
-            return root;
-        }
-
-        public async Task<HttpStatusCode> RecognizeAsync()
-        {
-            try
-            {
-                var requestJson = JSONProcessor.CreateInkRecognitionRequest(strokes.AsReadOnly());
-                var response = await httpManager.PutAsync(requestJson);
-                var statusCode = response.StatusCode;
-                if (statusCode == HttpStatusCode.OK)
-                {
-                    var responseJson = await response.Content.ReadAsStringAsync();
-                    root = JSONProcessor.ParseInkRecognizerResponse(responseJson);
-                }
-                return statusCode;
-            }
-            catch(Exception e)
-            {
-                throw;
-            }
-        }
-    }
-}
+﻿using Contoso.NoteTaker.Http;
+using Contoso.NoteTaker.JSON;
+using Contoso.NoteTaker.JSON.Format;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using Windows.Graphics.Display;
+using Windows.UI.Input.Inking;
+
+namespace Contoso.NoteTaker.Services.Ink
+{
+    public class InkRecognizer
+    {
+        List<InkRecognizerStroke> strokes;
+        InkRecognitionRoot root = null;
+
+        HttpManager httpManager;
+
+        // Default DPI setting

```

Comment:
perhaps you should given a reason why.

Reply:
There is comment explaining when to use default setting in L50-51 of this file

Reply:
[Wiki for DPI](https://en.wikipedia.org/wiki/Dots_per_inch) says Microsoft Windows use 96 as default DPI, [This blog](https://blogs.msdn.microsoft.com/fontblog/2005/11/08/where-does-96-dpi-come-from-in-windows/) also gives a explaination. But anyway, I think it is good enough to say it's windows default settings.

Reply:
@xinyzhou, the value of the default isn't the issue. The fact that the current dpi may be something else is. Someone looking at the code for the first time may wonder why you're hard coding the default. @gowthu2712 , please add a reference to the other comment.